### ISSUE TITLE
Multiple content parts on the same content type

### ISSUE THREAD

**Issue by gentledepp** [OrchardCMS/Orchard2#883]
Hi! It's me again - I hope I am not annoying you guys too much as you are doing a great work!

Back in the days when I was evaluating orchard v1 for a project, I hat the problem, that you can attach a content part to a content tye only once.

Supposed you have a content part "selected user" where a user can be picked that is e.g. assigned a task. It would be nice that you could add this part a second time (only with another name and configuration) to e.g. select a reviewer for a task.
That would be the same content part (type) on one content type.

But the content part instance of "assigned user" would have a different name obviously, and would display a different set of available users than the content part instance of "reviewer"

Is that a planned concept or completely out of scope?

(another example would be two "attachment parts" where one contains the documents that the assignee of a task could need to solve the task (like manuals, etc.) and the other part instance would be the documents that the assignee created by solving the task. (like a certification for an equipment that (s)he checked.  
That could allso mean that the assignee can only read the first attachment part instance (containing the manuals, etc.) but is allowed to edit the second part instance.)

**Comment by sfmskywalker** [comment]
Hey Marek! Although I'm a little bit fuzzy on the details, I believe this is exactly what you can do with Orchard Core today (v2)!

**Comment by gentledepp** [comment]
That would be great! Could you give me an example or some docs? :-)

**Comment by sfmskywalker** [comment]
I know @sebastienros demoed many months ago, so if you search through the [Orchard Project](http://orchardproject.net/) homepage, you should be able to find it. Alternatively, just install Orchard Core and run it from your local machine, then find the Content Type Definitions page, and see for yourself that you can attach multiple parts of the same type.

**Comment by sebastienros** [comment]
It's called Named Parts. In the Part settings, just check "Reusable". Then parts are named for a content type. Incidentally I tried it this morning for a specific part, and the editors don't show up anymore. Maybe it was broken recently, I am investigating right now.

**Comment by gentledepp** [comment]
Thank you! Named content parts it is :-)
